src/common/ui-components/Video/Video.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classNames from 'classnames/bind';
+import ReactPlayer from 'react-player';
+
+import myVideo from './Ahti_vertical.mp4';
+import styles from './Video.module.scss';
+
+const cx = classNames.bind(styles);
+
+export const Video: React.FC<any> = () => {
+  return (
+    <ReactPlayer
+      className={cx(styles.container)}
+      url={myVideo}
```

Comment:
Move video to `/public` and pass the url as a prop?

Reply:
This is not the final video, so I will do it later